Packs/AccentureCTI_Feed/ReleaseNotes/1_1_44.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### ACTI Indicator Feed
+
+- Fixed an issue where existing indicators were not fetched.
```

Comment:
Shouldn't it be more precise to say:
"Fixed an issue where existing indicators got falsy expired"?

Reply:
If that's the case, 'Fixed an issue where existing indicators were incorrectly expired.' ?